lnrpc/lightning.proto

```diff
@@ -597,6 +597,12 @@ service Lightning {
     */
     rpc AddAliases (AddAliasesRequest) returns (AddAliasesResponse);
 
+    /* lncli: `deletealiases`
```

Comment:
nit: there is no `lncli` command for this in the current PR, for both new RPCs. Should either remove this comment or add a CLI (otherwise the API generator script will fail because of this).

Reply:
was just carried by naive copy-paste. Removed it, don't think we need cli command for this